### DISCUSSION TITLE
Meaning of 总 in the context of jobs

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [rodrigo-morales-1/subtitles-npcr#1086] [Q&A]
What's the specific meaning of the character 总 in this context? Does it refer to a specific position inside the company (e.g. manager director, head of department, etc.)

https://github.com/rdrg109/subtitles-npcr/blob/b5d415b7f3aaa0fc4e31696621a845c1f0287a6a/sentences/45-text-1.yaml#L27-L31

@fhxshrs I think you should know this one. Please ping me when answering, so that I can get an email when you answer.

**Comment by fhxshrs** [comment]
@rdrg109 是的，“总”是“总经理”的简称。
